sqf-value/value.h

```diff
@@ -31,6 +31,8 @@
 #include <variant>
 #include <initializer_list>
 
+#undef assert
```

Comment:
assert macro is included elsewhere and conflicts with tester assert function

Reply:
this is the wrong place for that then tho, should be at the tests themself
not in the value header